packages/amplify-provider-awscloudformation/src/amplify-service-manager.js

```diff
@@ -415,6 +419,8 @@ async function uploadFile(s3, filePath, key) {
   }
 }
 
+const amplifyAppCreationEnabled = () => !process.env || process.env.CLI_DEV_INTERNAL_DISABLE_AMPLIFY_APP_CREATION !== '1';
```

Comment:
Why not boolean? Can you document the behavior of this env variable in the description please?

Reply:
0 and 1 is a common practice for env vars to represent booleans, in most of the cases code just checks if env var is present or not. I wanted to have a strict check on the value to reduce the possibility of misconfiguration.
I add more documentation to the PR description.